### ISSUE TITLE
Введение в Линукс

### ISSUE THREAD

**Issue by pavelzarubin** [fullstack-development/haskell-internship#10]
Предлагаю добавить базовый курс по Линуксу, в качестве дополнения к программе. Новичкам в любом случае придется ним начинать работать. 
https://stepik.org/course/73/promo
Курс немного устаревший, но рассказывает о некоторых базовых вещах (работа с терминалом, работа с архивами и файлами в интернете, vim, и т.д.)
Все упражнения в курсе выполняются прямо в степике и легко повторяются в Ubuntu.
Также можно добавить инструкцию, как поставить Ubuntu второй системой.
(например эту https://losst.ru/ustanovka-linux-ryadom-s-windows-10)

**Comment by antonkalinin-ml** [comment]
Спасибо, можно добавить, но только как вспомогательный материал. Мне не хочется обязывать всех учащихся ставить линукс, это на первых порах затрудняет обучение. Приходишь ты такой зеленый, неопытный, жаждешь учиться, а не можешь даже начать, пока не проделаешь гору телодвижений по установке того-сего. Может, этот хаскель человеку вообще не понравится, но это нельзя узнать, пока не начнешь учиться.

**Comment by antonkalinin-ml** [comment]
Спасибо, можно добавить, но только как вспомогательный материал. Мне не хочется обязывать всех учащихся ставить линукс, это на первых порах затрудняет обучение. Приходишь ты такой зеленый, неопытный, жаждешь учиться, а не можешь даже начать, пока не проделаешь гору телодвижений по установке того-сего. Может, этот хаскель человеку вообще не понравится, но это нельзя узнать, пока не начнешь учиться.

**Comment by antonkalinin-ml** [comment]
Хотя, по условию заданий надо использовать curl и писать шелл-скрипты, а ими пользоваться в винде затруднительно. Но это можно сделать в самом конце, когда сервер готов. Хотелось бы кривую обучения не делать слишком крутой, особенно в начале.

**Comment by antonkalinin-ml** [comment]
Хотя, по условию заданий надо использовать curl и писать шелл-скрипты, а ими пользоваться в винде затруднительно. Но это можно сделать в самом конце, когда сервер готов. Хотелось бы кривую обучения не делать слишком крутой, особенно в начале.

**Comment by stanislav-az** [comment]
Да, хороший курс, я сам его проходил когда-то. Стажерам придется разбираться либо сразу с линуксом, либо как поставить инструменты линукса на винду, по-моему лучше первое.
